src/bzfs/bzfsAPI.cxx

```diff
@@ -4286,10 +4286,29 @@ BZF_API const char* bz_getServerOwner()
     return getPublicOwner().c_str();
 }
 
+BZF_API void bz_reloadAll()
+{
+    bz_ReloadData_V1 event;
+    event.target = "all";
+    worldEventManager.callEvents(bz_eReloadEvent, &event);
```

Comment:
technically no, since this will send "all" and each call below will send a different string, but if "all" is exploded on the other side then ...

Reply:
Yea, I was thinking that if we want to listen to an explicit `bz_reloadAll()` or `/reload all`, it'd be useful to know when that happens even though it would trigger subsequent `bz_eReloadEvent`.